android-sdk/src/main/java/com/checkout/android_sdk/View/CardDetailsView.java

```diff
@@ -404,8 +404,10 @@ private void checkFullDate() {
                 mDataStore.getCardYear() != null &&
                 !CardUtils.isValidDate(mDataStore.getCardMonth(), mDataStore.getCardYear())) {
             mDataStore.setValidCardMonth(false);
-            ((TextView) mMonthInput.getSelectedView()).setError(getResources()
-                    .getString(R.string.error_expiration_date));
+           if(mMonthInput.getSelectedView()!=null){
+               ((TextView) mMonthInput.getSelectedView()).setError(getResources()
+                       .getString(R.string.error_expiration_date));
+           }
```

Comment:
Since the crash happens intermittently on Kotlin equivalent sample app. Can you please add a Kotlin equivalent sample app ?  which will can be used in future.